src/Docfx.Common/Docfx.Common.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
     <PackageReference Include="Spectre.Console" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
Add this reference to `Docfx.Common` project. 
Because JSON Schema validation logics are expected to be added this project.